tests/test_mrc.py

```diff
@@ -0,0 +1,110 @@
+import filecmp
+import os
+from datetime import datetime
+from unittest import TestCase
+
+import mrcfile
+import numpy as np
+
+from aspire.io.mrc import MrcStats
+
+
+class MrcStatsTestCase(TestCase):
+    def setUp(self):
+        # Create dummy data
+        self.n = n = 100
+        # Note, integers will overflow on windows, use floats.
+        self.a = a = np.arange(n * n).reshape((n, n)).astype(np.float32)
+
+        # Create stats instance
+        self.stats = MrcStats()
+
+        # Push chunks of data to it.
+        for i in range(n):
+            self.stats.push(a[i])
+
+    def testMin(self):
+        self.assertTrue(np.allclose(
+            np.min(self.a),
+            self.stats.amin))
+
+        self.stats.push(-10. * abs(self.stats.amin))
+        self.assertTrue(np.allclose(
+            -10. * abs(np.min(self.a)),
+            self.stats.amin))
+
+    def testMax(self):
+        self.assertTrue(np.allclose(
+            np.max(self.a),
+            self.stats.amax))
+
+        self.stats.push(10. * self.stats.amax)
+
+        self.assertTrue(np.allclose(
+            10. * np.max(self.a),
+            self.stats.amax))
+
+    def testMean(self):
+        self.assertTrue(np.allclose(
+            np.mean(self.a),
+            self.stats.amean))
+
+        orig = self.stats.amean
+
+        # Push some enough zeros to halve our original mean
+        self.stats.push(np.zeros(self.n * self.n))
+
+        self.assertTrue(np.allclose(
+            self.stats.amean,
+            orig/2.))
+
+        # Push enough mean data in to restore it
+        self.stats.push(np.ones(self.n * self.n) * orig * 2.)
+        self.assertTrue(np.allclose(
+            self.stats.amean,
+            orig))
+
+    def testRms(self):
+        self.assertTrue(np.allclose(
+            np.std(self.a),
+            self.stats.arms))
+
+    def testUpdate(self):
+        mrcs_filepath = 'test.mrc'
```

Comment:
Better to make sure these end up in a temp dir, no?

Reply:
Yes, that actually worked out a little cleaner. 41ef7fab44bc460fc52f04613ad62def6d62baea

Reply:
Nice. That's what I suspected. Can you add the same to `test_starfileio.py`?

Reply:
Sure. I went about it slightly differently there by stuffing it into the setup and breakdown because it is used multiple times, but has the same effect.

d46c9a5cc9207e02867ee080d3222725e97f34d9

Reply:
:+1: